tests/Microsoft.Identity.Test.Unit/PublicApiTests/ConfidentialClientApplicationTests.cs

```diff
@@ -2165,6 +2165,7 @@ public void ConfidentialClient_WithInvalidAuthority_ThrowsArgumentException()
         public async Task WithAccessTokenSha256ToRefresh_MatchingHash_GetsTokenFromIdp_Async()
         {
             const string accessToken = "access-token";
```

Comment:
Can we also add one more string to test ?
accessToken= `"test_token"` 
output -> `"cc0af97287543b65da2c7e1476426021826cab166f1e063ed012b855ff819656"`

Reply:
thanks @4gust,  updated to use, this matches the test data in the updated spec. 

```
accessToken= "test_token"
output -> "cc0af97287543b65da2c7e1476426021826cab166f1e063ed012b855ff819656"
```